v3ElectronicsUserGuide.md

```diff
@@ -735,19 +736,39 @@ cd /mnt/persistent/gemdaq/gemloader
     - Note this will kill any running process on the hardware, but if you're updating FW no one should be using the system anyway.
  31.  Summarize the actions you took in the elog entry you have already opened.
 
+## Preparing For a Power Cut
+
+In order to safely prepare a GEM test stand before a planned power cut execute:
+
+ 1. Power down the High Voltage.
+     - Use the opportunity to power down other sensitive equipement such as PMT's.
```

Comment:
Change to:

> Use the opportunity to power down other sensitive equipment, such as PMT's.

E.g. insert a comma after "equipment" and correct typo, e.g. equipment (correct) vs. equipement (wrong)